README.md

```diff
@@ -164,6 +164,25 @@ document.getElementById('logout').addEventListener('click', () => {
 });
 ```
 
+### Caching strategy
+
+The SDK can be configured to cache data either in memory or in local storage. The default is in memory. This setting can be controlled using the `cacheStrategy` option when creating the Auth0 client.
+
+To use the in-memory mode, no additional options need are required as this is the default setting. To configure the SDK to cache data using local storage, set `cacheStrategy` as follows:
+
+```js
+await createAuth0Client({
+  domain: '<AUTH0_DOMAIN>',
+  client_id: '<AUTH0_CLIENT_ID>',
+  redirect_uri: '<MY_CALLBACK_URL>',
+  cacheStrategy: 'localstorage' // valid values are: 'memory' or 'localstorage'
+}).then(auth0 => {
+  // ...
+});
+```
+
+**Important:** This feature will allow the caching of data **such as ID and access tokens** to be stored in local storage. Exercising this option changes the security characteristics of your application and **should not be used lightly**. Extra care should be taken to mitigate against XSS attacks and minimize the risk of tokens being stolen from local storage.
```

Comment:
I don't feel we've given users enough information here to make an informed decision. Is there anything we can link to?

Reply:
There will be.. security and docs are currently working together to come up with something that explains the concerns and the reasoning behind _why_ you might want to choose either local storage or memory and how it affects the security aspects of the application. The current documentation we could refer to here just says *don't store tokens in local storage*, which doesn't give the full picture.

It might be better to wait until that documentation surfaces and then we can link to that from here.

Reply:
Curious to read the full picture :)